src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -331,6 +331,9 @@
   - name: --api-server-authorized-ip-ranges
     type: string
     short-summary: Comma seperated list of authorized apiserver IP ranges. Set to 0.0.0.0/32 to restrict apiserver traffic to node pools.
+  - name: --enable-managed-identity
+    type: bool
+    short-summary: Using a system assigned managed identity to manage cluster resource group.
```

Comment:
is the managed identity for this specifc cluster for resource group?

Reply:
Yes. The cluster will get a system assigned identity, the identity will be used by this cluster's kube-controller-manager to managed resources in this cluster's "MC_" resource group.